Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -319,6 +317,13 @@ struct TranslationContextMenuInfo;
 #endif
 struct ViewportArguments;
 
+#if HAVE(DIGITAL_CREDENTIALS_UI)
+struct DigitalCredentialsRequestData;
+struct DigitalCredentialsResponseData;
+struct MobileDocumentRequest;
+struct OpenID4VPRequest;
```

Comment:
Why are these additional two structs needed?

Reply:
I'll remove this change for now...